css/style.css

```diff
@@ -75,3 +77,134 @@
   background-repeat: no-repeat;
   background-position: center;
 }
+
+/* WORKS SECTION */
+.section-works {
+  padding: 3.2rem 0;
+}
+
+.section-works .container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.secondary-heading {
+  font-size: 3.6rem;
+  font-weight: 700;
+  line-height: 1.3;
+  color: var(--main-color);
+  text-align: center;
+  margin-bottom: 3.2rem;
+}
+
+.section-works .separate {
+  margin: 3.2rem auto;
+  height: 5px;
+  width: 20%;
+  background-color: var(--secondary-color);
+  border: none;
+  border-radius: 5px;
+}
+
+.project {
+  border: 2px solid #ebf0ee;
+  border-radius: 10px;
+  text-align: center;
+  padding-bottom: 3.6rem;
+  margin-bottom: 3.6rem;
+}
+
+.project img {
+  width: 100%;
+  margin-bottom: 3.2rem;
+  overflow: hidden;
+}
+
+.project-heading {
+  font-size: 3rem;
+  font-weight: 600;
+  color: #3a4a42;
+  margin-bottom: 3.2rem;
+}
+
+.techs {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 4.8rem;
+}
+
+.techs-item {
+  font-size: 1.2rem;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  padding: 1.2rem;
+}
+
+.btn {
+  font-size: 2rem;
+  font-weight: 500;
+  text-decoration: none;
+  background-color: var(--secondary-color);
+  color: #fff;
+  padding: 1.2rem 2.4rem;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+/* WORKS SECTION */
+.about-header {
+  background-image: url("../images/illustration-\ about\ me-\ mobile.png");
```

Comment:
- here I used this background image that was exported from figma but it has one illustration
because  I didn't find one that has 2 illustration